reference/tools/user-guide.md

```diff
@@ -0,0 +1,189 @@
+---
+title: TiDB Ecosystem Tools User Guide
+category: reference
+aliases: ['/docs/dev/how-to/migrate/from-mysql/','/docs/dev/how-to/migrate/incrementally-from-mysql/','/docs/dev/how-to/migrate/overview/']
```

Comment:
@TomShawn dev need to be removed?

Reply:
I think `dev` has to be kept because the old links that users might have contains `dev`, so these aliases should also contain `dev`.

Reply:
@TomShawn Thanks. I will change the aliases in other release branches accordingly.

Reply:
Thanks!